### DISCUSSION TITLE
How to Stack / Append named ranges with hyperformula

### DISCUSSION THREAD

**Discussion by Keita2021** [handsontable/hyperformula#865] [General]
Good day.

I'm using hyperformula in glide tables for glide apps. Glide apps support google sheets but I'm migrating to using glide tables since the introduction of hyperformula in glide tables as I believe hyperformula will be a faster more effective calculation method. 

I was helped on google sheets forums to solve my problem but now I don't know how to do the same using hyperformula ( I have no idea of coding )

below is a link to the forum discussion that solved my problem in google sheets (please copy link a paste in new tab)

[https://docs.google.com/spreadsheets/d/1waUXTozSpPfoGjv6vvqWRDgjxBeP7I89QMZALWYGt0Q/edit#gid=911513338](https://docs.google.com/spreadsheets/d/1waUXTozSpPfoGjv6vvqWRDgjxBeP7I89QMZALWYGt0Q/edit#gid=911513338)

I need to get hyperformula to do the array lookup in this example. how would I write it as?

**Comment by AMBudnik** [comment]
Hi @Keita2021 

here's Aleksandra, Customer Success Manager at Handsontable (the organization under the HyperFormula project). Can I ask you to share your implementation details (what technology do you use and what you already build) with me by email at support@handsontable.com?

**Comment by warpech** [comment]
I think I found the missing forum link: https://support.google.com/docs/thread/125207655/how-to-stack-append-named-ranges?hl=en
